config/passport.js

```diff
@@ -98,36 +95,34 @@ module.exports = function (passport) {
     }));
 
   passport.use(new FacebookStrategy({
-
       clientID: configAuth.facebookAuth.clientID,
       clientSecret: configAuth.facebookAuth.clientSecret,
       callbackURL: configAuth.facebookAuth.callbackURL,
       profileFields: ["emails", "displayName", "gender"]
-    },
-
-    // facebook will send back the token and profile
-    function (token, refreshToken, profile, done) {
+    }, function(token, refreshToken, profile, done) {
       process.nextTick(function () {
-
         User.findOne({'facebook.id': profile.id}, function (err, user) {
           if (err)
             return done(err);
           if (user) {
             return done(null, user);
           } else {
             var newUser = new User();
-            newUser.facebook.id = profile.id;
-            newUser.facebook.token = token;
-            newUser.facebook.name = profile.displayName;
-            newUser.facebook.email = profile.emails[0].value;
+            newUser.facebook = {
+                id: profile.id,
+                token,
```

Comment:
This may need to be changed as it uses a[shorthand way to set props](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Object_initializer) and I'm not sure what version of node you're targeting this repo at. If you're going for v4+ you should be fine with this, if you're going for an older version then you'll need to change this to `token: token`.